### ISSUE TITLE
Display legend

### ISSUE THREAD

**Issue by miotatsu** [drones-for-ag/agronomist-toolbox#63]
depends on #57 (and #59?)

**Comment by miotatsu** [comment]
note to self: http://www.graphicsmagick.org/api/constitute.html#dispatchimage

**Comment by miotatsu** [comment]
reopening this because I can't get graphicsmagick to load pdfs on Windows. Currently experimenting with mupdf as an alternative

**Comment by miotatsu** [comment]
I can't get any pdf rendering libraries working on Windows at all but I did find a standalone imagemagick convert.exe binary. New plan is to spawn a process to convert pdf -> png and then load the png like anything else
